### ISSUE TITLE
Clearing Cache for util.prompt_for_user_token( )

### ISSUE THREAD

**Issue by rymc9384** [plamere/spotipy#163]
**TL;DR:** If you could have the ``util.prompt_for_user_token() `` function -- or the ``spotipy.Spotify(auth=token)`` function clear the cache created for authorization, that would be super helpful. Either removing the file entirely or being able to write over it w/o appending should work.

**Longer Version:**
I ran into an issue with getting the authorization to work, even though it had worked last night. 
I was getting an error from the *scope* argument about not being able to split a *None* type, even when *scope* was set as a string. The error came from the ``_is_scope_subset()`` function in **SpotifyOauth** class.

I realized that the cache from last night was still there, and upon removing it my code magically worked again. Suggested fix is discussed above in the **TL;DR** section.

Thanks for all your work!

**Comment by jonascj** [comment]
For anyone else wondering where this cache file is its alongside your python script called `.cache-username`, i.e. it is hidden on Linux. I missed it because I only did `ls -l` while looking for it.

**Comment by stephanebruckert** [comment]
Closing as wontfix as `util.prompt_for_user_token()` will be removed in v3
